scout/server/blueprints/managed_variants/templates/managed_variants/managed_variants.html

```diff
@@ -140,16 +140,28 @@ <h5>
 {% endmacro %}
 
 {% macro modify_variant_form(variant) %}
-  <form id="modify_variant" method="POST" action="{{ url_for('managed_variants.modify_managed_variant', variant_id=variant.managed_variant_id) }}" enctype="multipart/form-data" class="form-horizontal">
+  <form id="modify_variant_{{variant.variant_id}}" method="POST" action="{{ url_for('managed_variants.modify_managed_variant', variant_id=variant.managed_variant_id) }}" enctype="multipart/form-data" class="form-horizontal">
```

Comment:
This is require to modify the subcategory of the right variant when you have more than one managed var saved in the list

Reply:
Indeed!

Reply:
Oh, but lets remember to also loop over them in the `document.ready()` call as well - that´s how at least one of them got prepopulated before.. 🙄

Reply:
Ahh I see what you mean: this select should just have SNV and INDEL:

<img width="1148" alt="image" src="https://user-images.githubusercontent.com/28093618/194808063-a490903f-3a9a-43e0-8904-d0302a451ee7.png">

I'll fix!